### PR TITLE
Add PHP CS Fixer to composer file

### PR DESCRIPTION
**Pull request by vinkla**
Add the package to composer.json as introduced in #429

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
         "illuminate/filesystem": "5.0.*",
         "phpunit/phpunit": "~4.0",
         "mockery/mockery": "~0.9",
-        "squizlabs/php_codesniffer": "~2.0"
+        "squizlabs/php_codesniffer": "~2.0",
+        "fabpot/php-cs-fixer": "1.9.*"
```
Comment (lucasmichot) [inline composer.json]: No PHP-CS-Fixer should be installed globally.

File: composer.json [hunk 2]
```diff
@@ -46,5 +47,7 @@
         "branch-alias": {
             "dev-develop": "0.9-dev"
         }
-    }
+    },
+    "minimum-stability": "dev",
```
Comment (lucasmichot) [inline composer.json]: Not part of this feature

File: .php_cs [hunk 3]
```diff
@@ -5,5 +5,5 @@ $finder = Symfony\CS\Finder\DefaultFinder::create()
 
 return Symfony\CS\Config\Config::create()
     ->level(Symfony\CS\FixerInterface::PSR2_LEVEL)
-    ->fixers(array('short_array_syntax'))
+    ->fixers(['short_array_syntax'])
```
Comment (lucasmichot) [inline .php_cs]: No. PHP-CS-Fixer is still base on PHP >= 5.3.6
This should be long array syntax
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.9/composer.json#L17

File: composer.json (hunk 2 above)
Comment (vinkla) [inline composer.json]: It is if we want to use `1.9.*`.

File: composer.json (hunk 1 above)
Comment (GrahamCampbell) [inline composer.json]: Yeh, I agree.

File: composer.json (hunk 2 above)
Comment (GrahamCampbell) [inline composer.json]: No, we can just change the version constraint above to `1.9.*@dev`.

File: composer.json (hunk 1 above)
Comment (GrahamCampbell) [inline composer.json]: Why not just use https://styleci.io/ and ditch the cs checks from the tests. The tests and the cs should be separate. StyleCI will still use github's status API on pull requests.

File: composer.json (hunk 2 above)
Comment (vinkla) [inline composer.json]: Okay, I'll fix.

File: .php_cs (hunk 3 above)
Comment (GrahamCampbell) [inline .php_cs]: @lucasmichot So? We're only running php-cs-fixer on php 5.4+ in this context.

File: composer.json (hunk 1 above)
Comment (jasonlewis) [inline composer.json]: Easy to add in @GrahamCampbell?

Comment (vinkla) [inline composer.json]: @jasonlewis Just sign up at https://styleci.io/ and add the repository.

Comment (GrahamCampbell) [inline composer.json]: > Easy to add in @GrahamCampbell?

Yeh, it is, it at least I hope it is. Ping me if you need any help. :)

**Pull request closed without merging**